opencti-platform/opencti-front/src/private/components/common/files/workbench/WorkbenchFileCreator.tsx

```diff
@@ -130,6 +134,11 @@ const WorkbenchFileCreator: FunctionComponent<WorkbenchFileCreatorProps> = ({
               handleCloseCreate();
               onCompleted?.();
             },
+            onError: () => {
+              setSubmitting(false);
```

Comment:
As this code is shared between multiple places (onCompleted, onError, twice). I suggest to extract it into a "handleCompleted" or something like that method.